src/docs/constants/docs.constants.ts

```diff
@@ -155,6 +155,13 @@ export const COMPONENT_ROUTES: ComponentRoute[] = [
     description: "An indicator showing the completion progress of a task.",
   },
 
+  {
+    path: "/components/progress-steps",
+    title: "Progress Steps",
+    description:
+      "Display the progression of a long.lasting task with various steps.",
```

Comment:
Feel free to ignore suggestion.

```suggestion
      "Display the progression of a long-lasting task with various steps.",
```

Reply:
Thx